language/tools/move-cli/src/main.rs

```diff
@@ -136,22 +165,23 @@ pub enum Command {
         #[structopt(name = "file")]
         file: String,
     },
-    /// (Experimental) Run static analyses on Move source or bytecode
-    #[structopt(name = "analyze")]
-    Analyze {
-        #[structopt(subcommand)]
-        cmd: AnalysisCommand,
-    },
     /// Delete all resources, events, and modules stored on disk under `storage`.
     /// Does *not* delete anything in `src`.
     Clean {},
     /// Run well-formedness checks on the `storage` and `build` directories.
     #[structopt(name = "doctor")]
     Doctor {},
+    /// Type check and verify the scripts and/or modules under `src`.
+    #[structopt(name = "check")]
+    Check {
```

Comment:
Is this both a base command and a sandbox command?

Reply:
Yes. In the sandbox setting (`move sandbox check`) it will check the modules in `src` against the already published modules (and behaves just like before). The other base check (`move check`) must be given a list of files or directories and it will simply check those files. We may want to rename the command possibly? Happy to do so if you think it's confusing.

Reply:
Gotcha, that makes sense. We could perhaps rename the sandbox version something like `link`? But don't feel strongly, just seems a bit odd to have two commands with the same name that do (slightly) different things.

Reply:
Agreed that having the same command names meaning different things could be confusing. I'll change the name :)